proposals/lip-0041.md

```diff
@@ -170,6 +167,88 @@ This function verifies the nonce of a given transaction `trs` and returns a bool
 
 This function returns `true` if the transaction object contains a valid nonce, `false` otherwise. This is done using the `isValidNonce` function defined [below](#transaction-verification).
 
+### Genesis Block Processing
+
+The following two steps are executed as part of the genesis block processing.
```

Comment:
it is a bit weird to talk about both processing steps when there is only one which is relevant to this module.

Reply:
edited